setup.py

```diff
@@ -1,50 +1,45 @@
-import os
-from setuptools import setup, find_packages
+from distutils.core import setup
+import setuptools
+import sys
+from os.path import join
 
-here = os.path.abspath(os.path.dirname(__file__))
-
-requires = [
-    'dropbox',
-    'boxsdk',
-    'LAMP-core',
-    'paramiko',
-    'pyyaml',
-    'mano',
-    'yaxil',
-    'six',
-    'pytz',
-    'pandas',
-    'jsonpath_ng',
-    'dpanonymize@git+https://github.com/AMP-SCZ/dpanonymize.git'
-]
-
-test_requirements = [
-    'pytest',
-    'vcrpy'
-]
-
-about = dict()
-with open(os.path.join(here, 'lochness', '__version__.py'), 'r') as f:
```

Comment:
This approach was more advantageous than hard coding `version='v0.1.2'`. The latter is okay if the version string is not used anywhere else in the repository. Please check and confirm. 

If it is used elsewhere, then with your proposed setting, we shall have to update the version string in two places--once in `__version__.py` and then in `setup.py`.

Reply:
759defd